src/test/java/org/opensearch/knn/index/LuceneIT.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.index;
+
+import com.google.common.collect.ImmutableList;
+import org.opensearch.common.Strings;
+import org.opensearch.common.xcontent.XContentBuilder;
+import org.opensearch.common.xcontent.XContentFactory;
+import org.opensearch.knn.KNNRestTestCase;
+import org.opensearch.knn.common.KNNConstants;
+import org.opensearch.knn.index.util.KNNEngine;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class LuceneIT extends KNNRestTestCase {
```

Comment:
may be KNNUsingLuceneEngineIT?

Reply:
I'm not sure about that, I see similar tests for other engines follow this naming pattern - https://github.com/opensearch-project/k-NN/blob/main/src/test/java/org/opensearch/knn/index/NmslibIT.java or https://github.com/opensearch-project/k-NN/blob/main/src/test/java/org/opensearch/knn/index/FaissIT.java

Reply:
I don't think there is a pattern that we have to follow. Class name should convey meaning of it is own. I don't mind renaming other classes too :) but that is FAR. @jmazanec15 WDYT?

Reply:
I would probably call it LuceneEngineIT. Im not sure KNNUsing prefix is necessary.

Reply:
Yeah, just Lucene sounds too generic. I'll rename in one of next PRs.